### PR TITLE
Deploy version to display patch info

### PR DESCRIPTION
**Pull request by sselvara1**
This commit shall display deploy version with patch info.

Test Plan:
Passed: Deploy version to display base version
Passed: Deploy version to display base version and patch info

### CODE REVIEW

**Comment by yjian118** [comment]
What's the purpose of this commit? Doesn't the GitLastTag+GitBranch+GitHead not enough to indicate the deploy version?

**Review by hugonicodemos — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -19,9 +19,11 @@ HELM_CLIENT_VER_MAJ := $(shell echo ${HELM_CLIENT_VER} | awk -F. '{print $$2}')
 GIT_HEAD := $(shell git rev-list -1 HEAD)
 GIT_LAST_TAG := $(shell git describe --abbrev=0 --tags)
 GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
+GIT_PATCH := "1"
```
Comment (yjian118) [inline Makefile:22]: This GIT_PATCH should be "" on master branch, and we can update the patch in the stable branch

Comment (sselvara1) [reply Makefile]: GIT_PATCH is updated to "" on master branch.

**Review by yjian118 — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (sselvara1) [reply Makefile]: > This GIT_PATCH should be "" on master branch, and we can update the patch in the stable branch

to update patch in stable branch, should new jira be created?

**Pull request merged**